docs/developer-docs/integrations/ledger/deploy-new-token.md

```diff
@@ -6,7 +6,7 @@ This tutorial will guide you step-by-step to deploy your own token to the IC and
 
 1.  Ensure you have the ledger image, the private ledger interface, and the public ledger interface. If you do not have them, follow the steps in [Setup Ledger locally](./ledger-local-setup).
 
-2.  Ensure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on <https://smartcontracts.org/> to install it.
+2.  Ensure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on the [build](../../build/install-upgrade-remove) section to install it.
```

Comment:
I'm not sure "the [build] section" is correct here; I'd call it "the [install section]" or "the [Installing the SDK] section."

Reply:
totally agree - will change.